JS/Lesson19/saffet/exercises.js

```diff
@@ -1,8 +1,201 @@
-// 1. Create a Person class with name, age, and gender properties, and a greet() method that logs a greeting message to the console. Then create a Student class that extends the Person class and adds a major property. Override the greet() method in the Student class to log a different greeting message to the console. Create 2 variables, one of type Person and one of type Student and log them to console.
+// 1. Create a Person class with name, age, and gender properties, and a greet() method that logs a greeting message to the console. 
+// Then create a Student class that extends the Person class and adds a major property. Override the greet() method in the Student 
+// class to log a different greeting message to the console. Create 2 variables, one of type Person and one of type Student and 
+// log them to console.
+console.log("Example 1; ");
 
-// 2. Create a Shape class with name and color properties, and a describe() method that logs a description of the shape to the console (e.g. "This is a blue square"). Then create a Square class that extends the Shape class and adds sideLength and area properties. Override the describe() method in the Square class to log a description of the square to the console (e.g. "This is a blue square with a side length of 5 and an area of 25").
+class Person {
+    constructor(name, age, gender){
+        this.name = name;
```

Comment:
It's always better to use "private values inside a constructor, e.g. `this._name = name` and then in the `greet` method you'd use `this._name`

Reply:
I don't know if I understand correctly or not. But I hope it is.